src/subdomains/supporting/payin/strategies/register/impl/base/evm.strategy.ts

```diff
@@ -37,6 +42,7 @@ export abstract class EvmStrategy extends RegisterStrategy {
     protected readonly payInRepository: PayInRepository,
     protected readonly assetService: AssetService,
     private readonly repos: RepositoryFactory,
+    private readonly pricingService: PricingService,
   ) {
```

Comment:
You can use field injection instead of constructor injection (see e.g. AlchemyService) and remove the parameter from all strategies.